custom/icds_reports/ucr/data_sources/cbe_form.json

```diff
@@ -131,6 +131,181 @@
         "type": "expression",
         "column_id": "date_cbe_organise"
       },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "datatype": "string",
+          "type": "property_path",
+          "property_path": [
+            "form",
+            "cbe_info",
+            "theme_cbe"
+          ]
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "theme_cbe"
+      },
+      {
+        "column_id": "count_targeted_beneficiaries",
+        "datatype": "integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "root_doc",
```

Comment:
Is this necessary? Does it not work if you just replace this with the `property_path` below?

Reply:
Yeah you are right. It works. Thanks for catching it